js/src/carousel.js

```diff
@@ -150,6 +154,16 @@ class Carousel extends BaseComponent {
     }
     // End mod
 
+    // Boosted mod: if a play-pause button is present, set the button to play on mouseenter
+    if (this._playPauseButton !== null && this._playPauseButton.classList.contains('pause')) {
+      this._playPauseButton.classList.remove('pause')
+      this._playPauseButton.classList.add('play')
+      this._playPauseButton.setAttribute('title', 'Play Carousel')
```

Comment:
For this line and the one after, how can we change the text for non-english websites?